int32 frame_id
string curve_id
front_end/ProcTime[] processingTime
front_end/kPoint[] leftFeatures
front_end/kPoint[] rightFeatures
 
